samples-v2/orchestration_versioning/host.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      }
+    }
+  },
+  "extensions": {
+    "durableTask": {
+      "defaultVersion": "1.0"
```

Comment:
Can we add a comment here to highlight this is where customer set the Version feature?

Reply:
Are you literally talking about adding a comment to this JSON file? I don't think JSON officially supports comments. Some parsers _may_ support them, but this is non-standard.